README.md

```diff
@@ -1,5 +1,81 @@
-# tailwindcss-vertical-rhythm
+# Tailwindcss-vertical-rhythm
 
 A tailwind plugin which helps create scaling typography.
 
-TODO setup issues, readme, and maybe a better contributing md.
+# Installation
+You can install the library using your prefered package manager
+
+## NPM
+```sh
+npm install tailwindcss-vertical-rhythm
+```
+## Yarn
+```sh
+yarn add tailwindcss-vertical-rhythm
+```
+## PNPM
+```sh
+pnpm add tailwindcss-vertical-rhythm
+```
+# Configuration
+
+Add `tailwindcss-vertical-rhythm` as a plugin in `tailwind.config.js`
+
+```js
+module.exports = {
+  //further configurations
+  plugins: [require("tailwindcss-vertical-rhythm")]
+}
+```
+
+## Extracting Classes
+
+You can use a tailwind.css file for extracting classes and reuse code
+
+```css
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer components {
+  //name of class
+  .blue {
+    //css that will be applied
+    @apply py-2 px-4 bg-blue-500 text-white font-semibold rounded-lg shadow-md hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-400 focus:ring-opacity-75;
```

Comment:
The plugin generates some classes you can apply to text, the default ones created are:

`factor-xs`
`factor-sm`
`factor-base`
`factor-lg`
`factor-xl`
`factor-2xl`
`factor-3xl`
`factor-4xl`

I think it would make more sense to create an example class which uses some of these. Maybe a hero and content class? (Spit balling the component names here)

Reply:
I have made an example class and the css also uses the classes you mentioned